mongock-core/mongock-runner/mongock-runner-core/src/main/java/io/mongock/runner/core/internal/ChangeLogItem.java

```diff
@@ -101,7 +101,7 @@ public ChangeLogItem(String id,
                        List<CHANGESET> beforeChangeSetsItems,
                        boolean system) {
     if (id == null || id.trim().isEmpty()) {
-      throw new MongockException("id cannot be null or empty.");
+      throw new MongockException("ChangeUnit["type.getName() + "] id cannot be null or empty.");
```

Comment:
This line is not compiling, "+" is missing.

Reply:
Also, for coherence with previous exception, could you change "ChangeUnit" to "changeUnit"?

Reply:
Ok done.

Apologies for my laziness here - I'm trying to fix this code from the GitHub editor!